deployment/kubernetes/k8s/base/cluster-auth-token.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: stackstate-auth-token
+type: Opaque
+data:
+  # You can generate with: `echo -n '<ThirtyX2XcharactersXlongXtoken>' | base64`
+  token: PFRoaXJ0eVgyWGNoYXJhY3RlcnNYbG9uZ1h0b2tlbj4=
```

Comment:
could we override this token when applying manifests with kustomize ?

Reply:
Yes, I've introduced via SecretMapGenerator, but I think it also can be replaced as file fragment